docs/indyav/software/simulation.rst

```diff
@@ -0,0 +1,11 @@
+Simulation
+==========
+Vehicle and world models should go here. Along with any code that is needed for simulation to work.
```

Comment:
Remove `Vehicle and world models should go here. Along with any code that is needed for simulation to work.`